File: 3492-MaximumContainersonaShip/3492-MaximumContainersonaShip.js
// Last updated: 3/15/2026, 8:34:36 PM
1/**
2 * @param {number} n
3 * @param {number} w
4 * @param {number} maxWeight
5 * @return {number}
6 */
7var maxContainers = function(n, w, maxWeight) {
8    let value1 = n*n;
9    let value2 = Math.floor(maxWeight/w);
10
11    if(value1 < value2){
12        return value1;
13    }
14    else{
15    return value2;
16    }
17};

File: 3492-MaximumContainersonaShip/3492-MaximumContainersonaShip.cpp
// Last updated: 3/15/2026, 8:30:52 PM
1class Solution {
2public:
3    int maxContainers(int n, int w, int maxWeight) {
4       int value1 = n*n;
5       int value2 = maxWeight/w;
6
7       if(value1 < value2){
8        return value1;
9       } 
10       else{
11        return value2;
12       }
13    }
14};